File: python/Sequencial/EX5_A.py
#5.faca um programa que receba o salario de um funcionario e o percentual de aumento,calcule e mostre o aumento do salario.

print("********************SEQUENCIAL*************************")
print("********************Programa*5*************************")
print("*********Salário com percentual e aumento**************")
print("*******************************************************")

salario = float(input("Adicione o salario: "))
percentual = int(input("Adicione o percentual EX: 10 que é igual a 10%: "))

calc_acrec = salario * percentual/100

resultado = salario + (salario * percentual/100)

print("O aumento é R${: .2f}".format(resultado))

File: JS/Sequencial/EX5_A.js
/*
	5.faca um programa que receba o salario de um funcionario e o percentual de aumento,calcule e mostre o aumento do salario.
*/

confirm("Calculo do salario e o seu percentual")

var salario_base = prompt(" Adcione o salario ")
var percentual = prompt("Adicione o pecentual. Ex: 10 significa 10% ")

salario_base = parseFloat(salario_base)
percentual = parseInt(percentual)

resultado = salario_base + salario_base * percentual/100

console.log("O valor do salario atual é R$: "+ resultado)